### DISCUSSION TITLE
Como remover a página da Listas de Símbolos?

### DISCUSSION THREAD

**Discussion by leoneldca** [joaomlourenco/novathesis#193] [Q&A] [answered]
Olá, Boa tarde como eu devo proceder para remover a página da lista de símbolos, uma vez que a mesma está vazia?

**Comment by b-pereira** [comment, marked as answer]
Config/_ files.tex e mete a linha do ficheiro dos símbolos em comentário

**Reply by leoneldca** [reply]
Olá @b-pereira. Obrigado pela atenção

Eu tentei desta forma abaixomas sem sucesso:
Será que há alguma outra forma?


`%%------------------------------------------------------------
% Lists of Glossary, Acronyms and Symbols
%   syntax: \ntaddfile{glossaries}[list-type]{filename}
\ntaddfile{glossaries}[glossary]{glossary}
\ntaddfile{glossaries}[acronyms]{acronyms}
%\ntaddfile{glossaries}[symbols]{symbols}

%%------------------------------------------------------------
% Main text - document chapters
%   syntax: \ntaddfile{chapter}{filename}
%\ntaddfile{chapter}{symbols}
\ntaddfile{chapter}{chapter1}
\ntaddfile{chapter}{chapter2}
\ntaddfile{chapter}{chapter3}
\ntaddfile{chapter}{chapter4}
% \ntaddfile{chapter}{chapter-last}`

**Reply by b-pereira** [reply]
Eu fiz o que tu fizeste e funciona. Limpa a cache e tenta de novo.

![imagem](https://user-images.githubusercontent.com/944511/122232892-81f95100-ceb3-11eb-8e52-0395a60440f0.png)

**Reply by leoneldca** [reply]
Olá, eu fiz novamente conforme as orientações. Realmente compilou e gerou  pdf sem a página de símbolos.
Entretanto pra mim ( Uso o TextStudio ) aparece a mensagem que saiu com 
erros, mesmo tendo conseguido gerar o pdf. Talvez seja por eu não estar a
 usar o overleaf.
Segue abaixo o trecho do log de compilação

--------------------------------
NT LOADING “Chapters/acronyms” succeeded
! Missing endcsname inserted.

optk@/@nt/file/glossaries/symbols
l.196 \begin{document}
% Beginning of document
The control sequence marked  should
not appear between \csname and \endcsname.
! Extra \endcsname.
 .../file/glossaries/symbols \endcsname
l.196 \begin{document}
% Beginning of document
I'm ignoring this, since I wasn't doing a \csname.
! LaTeX Error: Missing \begin{document}.
See the LaTeX manual or LaTeX Companion for explanation.
Type  H   for immediate help.
...
l.196 \begin{document}
% Beginning of document
You're in trouble here.  Try typing    to proceed.
If that doesn't work, type  X   to quit.
NT LOADING “\optk@/@nt/file/glossaries/symbols ” failed
NT AtEndPreamble ntsetlayout

**Reply by leoneldca** [reply]
Consegui remover a página de Símbolos e já removendo-a do índice adicionando um * como no comando abaixo:

Antes
\ntaddfile{glossaries}[symbols]{symbols}

Depois
\ntaddfile{glossaries}[symbols]{symbols*}

Seu eu comentar a linha inteira ocorre o problema citado anteriormente.

Obigado!!!
